docs/cis_self_assessment.md

```diff
@@ -352,176 +349,97 @@ root:root
+Kubernetes makes use of a number of certificates as part of its operation. You should set the ownership of the directory containing the PKI information and all files in that directory to maintain their integrity. The directory and files should be owned by root:root. 
 </details>
 
 **Result:** Pass
 
-**Remediation:**
-Run the below command (based on the file location on your system) on the master node.
-For example,
-
-``` bash
-chown -R root:root /etc/kubernetes/ssl
-```
-
 **Audit:**
-
-```
-stat -c %U:%G /etc/kubernetes/ssl
+```bash
+stat -c %U:%G /var/lib/rancher/rke2/server/tls
+root:root
 ```
 
-**Expected result**:
+**Remediation:**
+By default, RKE2 creates the directory and files with the expected ownership of `root:root`. No manual remediation should be necessary.
 
-```
-'root:root' is present
+```bash
+chown -R root:root /var/lib/rancher/rke2/server/tls
 ```
 
 
 #### 1.1.20
 Ensure that the Kubernetes PKI certificate file permissions are set to `644` or more restrictive (Scored)
 <details>
 <summary>Rationale</summary>
-Kubernetes makes use of a number of certificate files as part of the operation of its components. The permissions on these files should be set to 644 or more restrictive to protect their integrity.
+Kubernetes makes use of a number of certificate files as part of the operation of its components. The permissions on these files should be set to 644 or more restrictive to protect their integrity. 
 </details>
 
 **Result:** Pass
 
-**Remediation:**
-Run the below command (based on the file location on your system) on the master node.
-For example,
+**Audit:** 
+Run the below command on the master node.
 
-``` bash
-chmod -R 644 /etc/kubernetes/ssl
-```
-
-**Audit Script:** check_files_permissions.sh
-
-```
-#!/usr/bin/env bash
-
-# This script is used to ensure the file permissions are set to 644 or
-# more restrictive for all files in a given directory or a wildcard
-# selection of files
-#
-# inputs:
-#   $1 = /full/path/to/directory or /path/to/fileswithpattern
-#                                   ex: !(*key).pem
-#
-#   $2 (optional) = permission (ex: 600)
-#
-# outputs:
-#   true/false
-
-# Turn on "extended glob" for use of '!' in wildcard
-shopt -s extglob
-
-# Turn off history to avoid surprises when using '!'
-set -H
-
-USER_INPUT=$1
-
-if [[ "${USER_INPUT}" == "" ]]; then
-  echo "false"
-  exit
-fi
-
-
-if [[ -d ${USER_INPUT} ]]; then
-  PATTERN="${USER_INPUT}/*"
-else
-  PATTERN="${USER_INPUT}"
-fi
-
-PERMISSION=""
-if [[ "$2" != "" ]]; then
-  PERMISSION=$2
-fi
-
-FILES_PERMISSIONS=$(stat -c %n\ %a ${PATTERN})
-
-while read -r fileInfo; do
-  p=$(echo ${fileInfo} | cut -d' ' -f2)
-
-  if [[ "${PERMISSION}" != "" ]]; then
-    if [[ "$p" != "${PERMISSION}" ]]; then
-      echo "false"
-      exit
-    fi
-  else
-    if [[ "$p" != "644" && "$p" != "640" && "$p" != "600" ]]; then
-      echo "false"
-      exit
-    fi
-  fi
-done <<< "${FILES_PERMISSIONS}"
+For example, 
```

Comment:
You do not need the "For example,"

the upstream doc has that because they are just giving an example command that would change based on what distribution of kubernetes you are testing. We know exactly what the command is. So, we don't need it.